embc-app/Authentication/SiteMinderAuthenticationToken.cs

```diff
@@ -45,9 +45,10 @@ public static SiteMinderAuthenticationToken GetFromHttpHeaders(HttpRequest req)
             };
         }
 
-        public static SiteMinderAuthenticationToken GetFromCookie(HttpRequest req)
+        public static SiteMinderAuthenticationToken CreateForDev(HttpRequest req)
         {
-            var str = req.Cookies[COOKIE_NAME]?.Base64Decode();
+            var str = req.Cookies[SM_TOKEN_NAME]?.Base64Decode();
+            if (string.IsNullOrEmpty(str)) str = req.Headers[SM_TOKEN_NAME].ToString().Base64Decode();
```

Comment:
Shouldn't we use the null check operator here in case the header returns null?
i.e. if (...) str = req.Headers[...]?.ToString()...

Reply:
Talked offline and the null check is not needed here